amazon_cloud_code_generator/api_specifications/AWS_Backup_BackupVault.json

```diff
@@ -0,0 +1 @@
+{"typeName": "AWS::Backup::BackupVault", "description": "Resource Type definition for AWS::Backup::BackupVault", "sourceUrl": "https://github.com/aws-cloudformation/aws-cloudformation-rpdk.git", "definitions": {"NotificationObjectType": {"type": "object", "additionalProperties": false, "properties": {"BackupVaultEvents": {"type": "array", "insertionOrder": false, "uniqueItems": false, "items": {"type": "string"}}, "SNSTopicArn": {"type": "string"}}, "required": ["SNSTopicArn", "BackupVaultEvents"]}, "LockConfigurationType": {"type": "object", "additionalProperties": false, "properties": {"MinRetentionDays": {"type": "integer"}, "MaxRetentionDays": {"type": "integer"}, "ChangeableForDays": {"type": "integer"}}, "required": ["MinRetentionDays"]}, "BackupVaultNamePattern": {"type": "string", "pattern": "^[a-zA-Z0-9\\-\\_]{2,50}$"}}, "properties": {"AccessPolicy": {"type": ["object", "string"]}, "BackupVaultName": {"$ref": "#/definitions/BackupVaultNamePattern"}, "BackupVaultTags": {"type": "object", "additionalProperties": false, "patternProperties": {"^.{1,128}$": {"type": "string"}}}, "EncryptionKeyArn": {"type": "string"}, "Notifications": {"$ref": "#/definitions/NotificationObjectType"}, "LockConfiguration": {"$ref": "#/definitions/LockConfigurationType"}, "BackupVaultArn": {"type": "string"}}, "additionalProperties": false, "taggable": true, "required": ["backup_vault_name"], "createOnlyProperties": ["backup_vault_name", "encryption_key_arn"], "writeOnlyProperties": ["changeable_for_days"], "readOnlyProperties": ["backup_vault_arn"], "primaryIdentifier": ["backup_vault_name"], "handlers": {"create": {"permissions": ["backup:CreateBackupVault", "backup:PutBackupVaultAccessPolicy", "backup:PutBackupVaultNotifications", "backup:PutBackupVaultLockConfiguration", "backup-storage:Mount", "backup-storage:MountCapsule", "kms:CreateGrant", "kms:GenerateDataKey", "kms:Decrypt", "kms:RetireGrant", "kms:DescribeKey"]}, "read": {"permissions": ["backup:DescribeBackupVault", "backup:GetBackupVaultNotifications", "backup:GetBackupVaultAccessPolicy", "backup:ListTags"]}, "update": {"permissions": ["backup:DescribeBackupVault", "backup:DeleteBackupVaultAccessPolicy", "backup:DeleteBackupVaultNotifications", "backup:DeleteBackupVaultLockConfiguration", "backup:ListTags", "backup:TagResource", "backup:UntagResource", "backup:PutBackupVaultAccessPolicy", "backup:PutBackupVaultNotifications", "backup:PutBackupVaultLockConfiguration"]}, "delete": {"permissions": ["backup:DeleteBackupVault"]}, "list": {"permissions": ["backup:ListBackupVaults"]}}}
```

Comment:
I guess you should also delete the old api specifications because we do not need them anymore.

Reply:
The files have been rebased and renamed. I am not sure why it is not shown. I would like to merge the PR and see the result.

Reply:
Also, it seems to me that this PR doesn't respect anymore the indentation as you can see there https://github.com/ansible-collections/amazon_cloud_code_generator/commit/bc841d540b4424d1dd312fc93a8510ca6bffc53e.

Reply:
I guess there was an issue with your rebase.